.codecov.yml

```
@@ -107,6 +107,10 @@ coverage:
         target: 75
         flags:
         - reboot_required
+      RedisEnterprise:
```

```suggestion
      Redis_Enterprise:
```
CI is failing on this